ksqldb-parser/src/test/java/io/confluent/ksql/parser/json/KsqlTypesSerdeModuleTest.java

```diff
@@ -75,19 +78,22 @@ public void shouldSerDeSqlArrayTypes() throws JsonProcessingException {
   @Test
   public void shouldSerDeSqlMapTypes() throws JsonProcessingException {
     // Given:
-    final SqlType[] types = new SqlType[]{
+    final List<SqlType> types = ImmutableList.of(
         SqlTypes.INTEGER,
         SqlTypes.BIGINT,
         SqlTypes.DOUBLE,
         SqlTypes.STRING
-    };
+    );
 
-    for (final SqlType type : types) {
-      // When:
-      final SqlType out = MAPPER.readValue(MAPPER.writeValueAsString(SqlMap.of(type)), SqlType.class);
+    for (final SqlType keyType : types) {
+      for (final SqlType valueType : Lists.reverse(types)) {
```

Comment:
Any particular reason this is reversed?

Reply:
So they don't match.  Otherwise the code could be flipping key and value and the test would still pass.

Reply:
I get why the additional for-loop was added (for that reason), but I'm wondering by there's `Lists.reverse()` in there. Is it because Guava contains a convenient method for copying a list and reversing its contents, but not for directly copying the list...?